index.html

```diff
@@ -1418,6 +1461,9 @@
           </ul>
         </li>
         <li>
+          Let <var>options</var> be the second argument.
```

Comment:
You don't need this, since you've introduced it with `pushMessage(message, options)` above.